src/dotnet/APIView/APIViewWeb/Models/ReviewModel.cs

```diff
@@ -76,5 +76,8 @@ public string DisplayName
 
         [JsonIgnore]
         public string Language => Revisions.FirstOrDefault()?.Files.FirstOrDefault()?.Language;
+
+        // Master version of review for each package will be locked
+        public bool IsLocked { get; set; }
```

Comment:
Let's call it more descriptive, IsAutomatic? IsReadonly?

Reply:
I will rename to isAutomatic